tmt/steps/provision/__init__.py

```diff
@@ -289,7 +289,9 @@ def _ansible_playbook_path(self, playbook: str) -> str:
         assert parent.plan.my_run is not None  # narrow type
         assert parent.plan.my_run.tree is not None  # narrow type
         assert parent.plan.my_run.tree.root is not None  # narrow type
-        playbook = os.path.join(parent.plan.my_run.tree.root, playbook)
+        # TODO: remove once pathlib transition is complete
+        assert not playbook.is_absolute()
```

Comment:
Shouldn't we rather remove the leading slash if present?

Reply:
Hmm, maybe. I don't see the leading slash removed in the original code, `os.path.join` would also fail on this. But I can't dispute your logic.

Reply:
Attempting to solve it in d6efd994

Reply:
Thanks, `Path.unrooted()` will definitely be useful.